### PR TITLE
Update intro act after testing at opek 28/11/2016

### PR DESCRIPTION
**Pull request by hoangtungdinh**
Mainly update Nerver and Dumbo intro act

### CODE REVIEW

File: trajectory/src/main/java/io/github/agentwise/swarmview/trajectory/applications/trajectory/StraightLineTrajectory4D.java [hunk 1]
```diff
@@ -72,10 +72,10 @@ private StraightLineTrajectory4D(
         checkArgument(
                 velocityCutoffTimePercentage <= 1 && velocityCutoffTimePercentage > 0,
                 "Velocity cutoff percentage should represent a percantage between 0 and 1.");
-        checkArgument(
-                velocity <= MAX_ABSOLUTE_VELOCITY,
-                "The provided enterVelocity should be smaller than BasicTrajectory"
-                        + ".MAX_ABSOLUTE_VELOCITY");
+//        checkArgument(
```
Comment (mhct) [inline trajectory/src/main/java/io/github/agentwise/swarmview/trajectory/applications/trajectory/StraightLineTrajectory4D.java:75]: Why did you remove the checkArgument for the velocity of the StraightLineTrajectory4D? 

Removing this check may lead to code that tries to make the drone move faster than its allowed maximum speed.

Comment (hoangtungdinh) [reply trajectory/src/main/java/io/github/agentwise/swarmview/trajectory/applications/trajectory/StraightLineTrajectory4D.java:75]: It's for the wiggle effect. We need to generate very high velocity (15m/s) for the wiggle.

Comment (mhct) [reply trajectory/src/main/java/io/github/agentwise/swarmview/trajectory/applications/trajectory/StraightLineTrajectory4D.java:75]: Ok. I will merge and create an issue requesting to add  a constructor or builder that **explicitly** mentions the lack of maximum speed check when using it.
